ts/screens/wallet/payment/ScanQrCodeScreen.tsx

```diff
@@ -156,14 +150,6 @@ class ScanQrCodeScreen extends React.Component<Props, State> {
 
   private handleWillBlur = () => this.setState({ isFocused: false });
 
-  private openAppSettings = () => {
-    if (Platform.OS === "ios") {
-      Linking.openURL("app-settings://notification/IO").catch(_ => undefined);
```

Comment:
You have replaced  ""app-settings://notification/IO" with "app-settings:". I this intentional?

Reply:
@fpersico 
yes it is. I guess that code was also confusing because **app-settings://notification/IO** asks to open the **notification** settings of the IO app (that code opens the app settings anyway). Indeed the ScanQrCodeScreen has to ask to open the details app settings.

I tested the previous version and the actual and the behaviour is the same.

If you want I can restore the 3 differents functions